db/migrate/20240109223006_create_posts.rb

```diff
@@ -0,0 +1,13 @@
+class CreatePosts < ActiveRecord::Migration[7.1]
+  def change
+    create_table :posts do |t|
+      t.integer :author_id
```

Comment:
👍 Good job on creating the migration file for the `posts` table! However, there's a crucial element missing in the current setup.

- [ ] Upon reviewing the database diagram, it's evident that the `posts` table should have a one-to-many relationship with the `users` table. This association is essential for linking posts to their respective authors. To address this, please follow the [official documentation](https://guides.rubyonrails.org/active_record_migrations.html#creating-associations-using-references) on creating associations using references. Update your migration file accordingly to establish the relationship between the `posts` and `users` tables using `author_id`. After making the necessary changes, run the migration files to ensure that the code works successfully.
![image](https://github.com/francksefu/blog_web/assets/55840999/5ac8cd16-6aa8-472a-bdc3-673499041760)

Keep up the good work! If you have any questions or need further guidance, feel free to ask. 🚀👍

Reply:
Thanks for the review, but as you know, when we put the belongs_to in migration, it's just add  tablename_id as a new column, and the proper association is set in model, so that why I decided to create all the foreign column manually and set the association in the column ... is it bad to do that ?? @shahadat3669